lib/tasks/populate_threshold_sets.rake

```diff
@@ -0,0 +1,18 @@
+namespace :threshold_sets do
```

Comment:
How are we going to mark default sets?

Reply:
I think we can do this manually after it's implemented.